cmd/chart.go

```diff
@@ -112,12 +112,16 @@ func moveChart(cmd *cobra.Command, args []string) error {
 		Source: mover.Source{
 			Chart:          mover.ChartSpec{},
 			ImageHintsFile: imagePatternsFile,
+			// Use local keychain for authentication
```

Comment:
note that currently relok8s do not support the use of credentials while its library does